src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.stories.tsx

```diff
@@ -52,3 +53,21 @@ export const withOneTabs = () => {
     );
 };
 withOneTabs.story = { name: 'with one tab' };
+
+export const withNoMostSharedImage = () => {
+    fetchMock.restore().getOnce('*', {
+        status: 200,
+        body: responseWithMissingImage,
+    });
+
+    return (
+        <Section>
+            <MostViewedFooter
+                pillar="news"
+                ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+            />
+            ;
```

Comment:
I assume this is an accidental `;`

Reply:
Random. No idea where they came from. Fixed now.